db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml

```diff
@@ -8964,6 +8964,19 @@ Shutting down instance {0} on database directory {1} with class loader {2} </tex
            </msg>
 
 
+           <msg>
+               <name>EXT21</name>
+               <text>'{0} is not defined in the table definition'</text>
+               <arg>columnName</arg>
+           </msg>
+
+           <msg>
+               <name>EXT22</name>
+               <text>'Partition column {0} need to be numerical '</text>
```

Comment:
'numerical' isn't the same thing as 'scalar'; a float is numerical but is not a scalar.